testing/trino-tests/pom.xml

```diff
@@ -208,13 +208,27 @@
             <groupId>io.trino</groupId>
             <artifactId>trino-iceberg</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <!-- conflicts with com.google.protobuf:protobuf-java -->
+                    <groupId>org.apache.iceberg</groupId>
+                    <artifactId>iceberg-orc</artifactId>
```

Comment:
why do you think it's safe to exclude iceberg-orc here? it looks like quite "a core" component of iceberg?